### DISCUSSION TITLE
Contaminants annotation

### DISCUSSION THREAD

**Discussion by MarST07** [vdemichev/DiaNN#371] [General]
Hello Vadim,

I thank you for the great tool DIA-NN! I am very impressed of these great results and the easy Output of the data, as well as the possibility to process the data in R.
I have one question regarding contaminant annotation of the proteins in the proteingroups text file. Is it possible to implement these information out of the MaxQuant Contaminant database?

Kind Regards,
Stella

**Comment by vdemichev** [comment]
Hi Stella,

Sorry, not familiar with MaxQuant. DIA-NN will annotate things in whatever way they are annotated in the FASTA database(s) you supply to it. So all you need to do is to supply a contaminant FASTA (in addition to the main database) with whatever annotaions you like.

Best,
Vadim

**Reply by MarST07** [reply]
Thank you for the quick response!
